frontend/src/components/resultPage/resultsReferredOut/ReferredOutTests.js

```diff
@@ -0,0 +1,564 @@
+                  <Button type="button" onClick={handleReferredOutPatient}>
+                    <FormattedMessage
+                      id="referral.main.button"
+                      defaultMessage="Search Referrals By Patient"
+                    />
+                  </Button>
+                </div>
+              </div>
+            </Section>
+          </Column>
+        </Grid>
+        <hr />
+        <Formik
+          initialValues={referredOutTestsFormValues}
+          enableReinitialize={true}
+          // validationSchema={}
+          onSubmit={handleSubmit}
+          onChange
+        >
+          {({
+            values,
+            //errors,
+            //touched,
+            setFieldValue,
+            handleChange,
+            handleBlur,
+            handleSubmit,
+          }) => (
+            <Form
+              onSubmit={handleSubmit}
+              onChange={handleChange}
+              onBlur={handleBlur}
+            >
+              <Field name="guid">
+                {({ field }) => (
+                  <input type="hidden" name={field.name} id={field.name} />
+                )}
+              </Field>
+              <Grid fullWidth={true}>
+                <Column lg={16} md={8} sm={4}>
+                  <Section>
+                    <div className="inlineDiv">
+                      <h5 style={{ paddingTop: "10px", paddingRight: "6px" }}>
+                        <FormattedMessage id="referral.out.request" />
+                      </h5>
+                      <Dropdown
+                        id={"dateType"}
+                        name="dateType"
+                        label="Date Type"
+                        initialSelectedItem={dateTypeList.find(
+                          (item) => item.tag === dateType,
+                        )}
+                        items={dateTypeList}
+                        itemToString={(item) => (item ? item.text : "")}
+                        onChange={(item) => {
+                          setSearchType(
+                            referredOutTestsFormValues.searchTypeValues[0],
+                          );
+                          setDateType(item.selectedItem.tag);
+                        }}
+                      />
+                      <h5 style={{ paddingTop: "10px", paddingLeft: "6px" }}>
+                        <FormattedMessage id="referral.out.note" />
+                      </h5>
+                    </div>
+                    <div className="formInlineDiv">
+                      <CustomDatePicker
+                        id={"startDate"}
+                        labelText={intl.formatMessage({
+                          id: "eorder.date.start",
+                          defaultMessage: "Start Date",
+                        })}
+                        autofillDate={true}
+                        value={referredOutTestsFormValues.startDate}
+                        className="inputDate"
```

Comment:
When you use the carbon Grid and Column , remove all these custom css classes on the input components